### PR TITLE
Enable dimension handler

### PR DESCRIPTION
**Pull request by eSlider**
There is an Dimension Handler realised in some local project, avaible as source code, but never enabled. I suggest enable it in `release/3.0.7`

### CODE REVIEW

File: src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js [hunk 1]
```diff
@@ -956,10 +956,28 @@
                         lastItem.after(chkbox);
                         var inpchkbox = chkbox.find('.checkbox');
                         inpchkbox.data('dimension', item);
+                        initCheckbox.call(inpchkbox);
```
Comment (atd-schubert) [inline src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js]: Where is this global function from? Can't we require or import this in a "normal" way?

We use this global function also in:
* application-edit.js
* mapbender.element.wmceditor.js

Comment (LazerTiberius) [reply src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js]: This function is defined in FOM/CoreBundle/Resources/public/js/widgets/checkbox.js in the global scope. In my opinion, we should move this to some kind of utility object or something similar. Defining something in the global scope without a namespace is always bad.

Comment (LazerTiberius) [reply src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js]: I created an issue regarding this global ui helper functions https://github.com/mapbender/mapbender/issues/615

**Comment by werrolf** [comment]
NTS1: if this doesn't meet acceptance (documentation, usability), we can trivially un-expose the element with a single-line change in src/Mapbender/WmsBundle/MapbenderWmsBundle.php
NTS2: if we do accept we need a changelog entry

**Pull request merged**